.gitignore

```diff
@@ -96,3 +99,14 @@ dump.rdb
 
 # Visual Studio Code
 .vscode
+
+.idea
+*-export
```

Comment:
`*-export` and `/new_service` are specific to teacher-services-cloud repo, I'm not sure they're relevant here

Reply:
i have removed them